src/translations/nbTranslations.json

```diff
@@ -573,6 +575,8 @@
     },
     "project_registrations": "Prosjektregistreringer",
     "registrations": {
+      "delete_all_draft_registrations": "Slett alle kladder",
+      "delete_all_draft_registrations_message": "Slett alle upubliserte registreringer?",
```

Comment:
Har noen sagt noe om naming av "kladder" vs "upliserte registreringer"? Ser det brukes "kladd" på de 3 andre nye oversttelsene, og villa antatt vi ønsker å bruke samme uttrykk overalt for dette, men idk

Reply:
Det ble sagt i ukentlig designmøte 16.08 at "kladd" var ønskelig